static/js/portico/landing-page.js

```diff
@@ -130,6 +131,12 @@ var apps_events = function () {
             $third_party_apps.hide();
         }
 
+        if (version === "android") {
```

Comment:
In cases like this you can use jQuery's `toggle` function that shows or hides an element based on a boolean value.